src/coreComponents/mesh/generators/CellBlockManagerABC.hpp

```diff
@@ -101,12 +102,20 @@ class CellBlockManagerABC : public dataRepository::Group
   virtual Group & getFaceBlocks() = 0;
 
   /**
-   * @brief Returns a group containing the well blocks as @p WellBlockABC instances.
+   * @brief Returns a group containing the well blocks as @p LineBlockABC instances.
    * @return Mutable reference to the well blocks group.
    *
    * @note It should probably be better not to expose a non-const accessor here.
    */
-  virtual Group & getWellBlocks() = 0;
+  virtual Group & getLineBlocks() = 0;
```

Comment:
If we need `getLineBlocks` for the `CellBlockManager`, do we need it for the `ABC` version?

Reply:
Indeed it has been removed, dont remember when, too much hollidays ^^...